### DISCUSSION TITLE
MODBUS RTU via RS485 ports of inverter

### DISCUSSION THREAD

**Discussion by TheFlyingSquegee** [celsworth/lxp-bridge#275] [General]
Lxp-bridge has been a great tool for me to use. The only catch is that it takes around 2 minutes to get data. I notice on my SNA5000 there are RS485 ports, so can I use that to read the registers value? I have been using a FT232 USB-2-RS485 and a serial monitor software to sniff the ports but it doesn't work. Any help would be appreciated!

**Comment by nickjf20** [comment]
Use the 'INV485' port. You will also need to disconnect the WiFi dongle.

Commercial solution 'coming soon':
https://diysolarforum.com/threads/eg4-6000xp-and-solarassistant.76181/page-3

Python RS485-MQTT:
https://github.com/HotNoob/PythonProtocolGateway/wiki/Devices%5CEG4-to-MQTT

**Reply by TheFlyingSquegee** [reply]
Thank you for your suggestion. I'll give it a go and see if it works.

**Comment by XanhLa96** [comment]
Everyone, Who try this? It works??? Hm?
